genotype/store/trending.py

```diff
@@ -0,0 +1,61 @@
+"""Functions to create trending specific data."""
```

Comment:
code to create trending specific data should be move to cg, our integration package, right?